MathJax.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+class MathJax extends MathRenderer {
```

Comment:
I'd prefer "MathJaxServer", as there already is/was a "MathJax" mode for client side